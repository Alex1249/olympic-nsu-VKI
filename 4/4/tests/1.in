14
0 100
2 30 1
2 31 1
2 32 1
1 31
1 7
2 31 0
1 31
3 30 33
3 31 32
3 0 100
3 45 67
0 48
1 30
