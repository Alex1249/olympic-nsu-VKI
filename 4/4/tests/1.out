1
0
0
some
none
some
none
0
